### PR TITLE
Add circle layers

### PR DESCRIPTION
**Pull request by KipSigei**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "jsx-loader": "^0.13.2",
     "moment": "^2.22.1",
     "mustache": "^2.3.0",
+    "neo-async": "^2.5.1",
```
Comment (cKellyDesign) [inline package.json]: @KipSigei Wait what's this?

File: src/connectors/ona-api/aggregateFormData.js [hunk 2]
```diff
@@ -113,6 +119,8 @@ function processFormData(formData, indicatorField, aggregateOptions) {
       .map(p => p.p);
   }
 
+  // let
```
Comment (cKellyDesign) [inline src/connectors/ona-api/aggregateFormData.js]: Let's remove this

File: src/connectors/ona-api/aggregateFormData.js [hunk 3]
```diff
@@ -185,15 +214,21 @@ function processFormData(formData, indicatorField, aggregateOptions) {
...
 
           if (extraProps && extraProps.length) {
-            extraProps.forEach((e, y) => {
-              extraPropsSumTotal[y] += parseInt(groupData[x][e] || 0, 10);
-            });
+            for (let y = 0; y < extraProps.length; y += 1) {
+              const e = extraProps[y];
+              if (numberProps.indexOf(e) !== -1) {
+                const int = parseInt(groupData[x][e] || 0, 10);
+                extraPropsSumTotal[e] += int;
```
Comment (cKellyDesign) [inline src/connectors/ona-api/aggregateFormData.js]: We can collapse this into one line and remove the `const inst`, that was just for debugging

File: package.json (hunk 1 above)
Comment (KipSigei) [reply package.json]: not really sure, I got a `missing 'neo-async' module` error when compiling so I added it

File: src/connectors/ona-api/aggregateFormData.js (hunk 2 above)
Comment (KipSigei) [reply src/connectors/ona-api/aggregateFormData.js]: addressed

File: src/connectors/ona-api/aggregateFormData.js (hunk 3 above)
Comment (KipSigei) [reply src/connectors/ona-api/aggregateFormData.js]: addressed

**Pull request merged**